README.md

```diff
@@ -570,6 +577,52 @@ const tokenboundClient = new TokenboundClient({ walletClient, chainId: 1, versio
 
 ---
 
+### Appending Calls To Account Creation
+
+You can make your first transaction using a newly-created TBA by appending a call to `createAccount`'s internal multicall sequence for execution after the account creation and initialization steps. To determine your account address before it has been created, use the `getAccount` method. You can then use the account as the Call3 `target`.
+
+See the [Multicall3 docs](https://github.com/mds1/multicall) for detailed info regarding this approach. Pay special attention to the notices re: contract writes.
+
+
+```typescript
+import { Call3 } from '@tokenbound/sdk'
+import { encodeFunctionData } from 'viem'
+
+const TOKENBOUND_ACCOUNT_ADDRESS = `0xABC123...`
+
+// Encode function data for use in prepareExecution call
+const encodedFunctionData = encodeFunctionData({
```

Comment:
Could we use an example that shows execution from the TBA here? I imagine that will be the most common use case of this feature that folks will want to use.

Reply:
Done.